doc/api/n-api.md

```diff
@@ -4612,7 +4612,7 @@ napi_create_threadsafe_function(napi_env env,
 ```
 
 - `[in] env`: The environment that the API is invoked under.
-- `[in] func`: The JavaScript function to call from another thread.
+- `[in] func`: Optional JavaScript function to call from another thread.
```

Comment:
Can you add a `changes:` entry to the YAML metadata above?

Reply:
Which version are we going to mark the changes? Would it be ok with v10.16.0?